metals/src/main/scala/scala/meta/internal/metals/debug/server/DebugeeParamsCreator.scala

```diff
@@ -0,0 +1,138 @@
+package scala.meta.internal.metals.debug.server
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.concurrent.Promise
+
+import scala.meta.internal.metals.BuildTargets
+import scala.meta.internal.metals.JavaTarget
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals.ScalaTarget
+
+import ch.epfl.scala.bsp4j.BuildTargetIdentifier
+import ch.epfl.scala.bsp4j.MavenDependencyModule
+import ch.epfl.scala.debugadapter.Library
+import ch.epfl.scala.debugadapter.Module
+import ch.epfl.scala.debugadapter.ScalaVersion
+import ch.epfl.scala.debugadapter.SourceDirectory
+import ch.epfl.scala.debugadapter.SourceJar
+import ch.epfl.scala.debugadapter.StandaloneSourceFile
+import ch.epfl.scala.debugadapter.UnmanagedEntry
+
+class DebugeeParamsCreator(buildTargets: BuildTargets)(implicit
+    ec: ExecutionContext
+) {
+  def create(
+      id: BuildTargetIdentifier,
+      cancelPromise: Promise[Unit],
+  ): Option[Future[DebugeeProject]] = {
+    val optScalaTarget = buildTargets.scalaTarget(id)
+    val optJavaTarget = buildTargets.javaTarget(id)
+    for {
+      name <- optScalaTarget
+        .map(_.displayName)
+        .orElse(optJavaTarget.map(_.displayName))
```

Comment:
```suggestion
      name <- buildTargets.jvmTarget(id).map(_.displayName))
```

and add displayName to JvmTarget trait?